src/python/pants/option/global_options.py

```diff
@@ -36,6 +37,8 @@ def register_bootstrap_options(cls, register):
     status as "bootstrap options" is only pertinent during option registration.
     """
     buildroot = get_buildroot()
+    default_distdir = os.path.join(buildroot, 'dist')
+    default_rel_distdir = '/{}/'.format(fast_relpath(default_distdir, get_buildroot()))
```

Comment:
This is computed a few lines above, so using `fast_relpath` to tear it back apart doesn't make sense.

Reply:
fixed